### PR TITLE
Fix/article comment bug

### PR DESCRIPTION
**Pull request by pitb2022**
https://www.notion.so/matterslab/Zhao-Nan-9bffdb932e32435bbd799ae742886c9e

### CODE REVIEW

File: src/components/Dialogs/CommentFormBetaDialog/CommentForm/index.tsx [hunk 1]
```diff
@@ -61,6 +68,8 @@ const CommentForm: React.FC<CommentFormProps> = ({
   }:${replyToId || 0}`
   const formId = `comment-form-${commentDraftId}`
   const formRef = useRef<HTMLFormElement>(null)
+  const { getQuery, router, routerLang } = useRoute()
+  const mediaHash = getQuery('mediaHash')
```
Comment (robertu7) [inline src/components/Dialogs/CommentFormBetaDialog/CommentForm/index.tsx:72]: beware of the breaking change by the new article URL

File: src/components/Dialogs/CommentFormBetaDialog/CommentForm/styles.module.css [hunk 2]
```diff
@@ -10,6 +10,18 @@
...
   overflow-y: auto;
+
+  & > div {
+    height: 100%;
+
+    & > div {
+      height: 100%;
+
+      & > div {
+        height: 100%;
```
Comment (robertu7) [inline src/components/Dialogs/CommentFormBetaDialog/CommentForm/styles.module.css:21]: it's not a good practice to select elements with nested selectors, is there any other solution (e.g. class name)?

**Review by robertu7 — APPROVED** [review]

File: src/components/Dialogs/CommentFormBetaDialog/CommentForm/index.tsx (hunk 1 above)
Comment (pitb2022) [reply src/components/Dialogs/CommentFormBetaDialog/CommentForm/index.tsx:73]: TODO note has been added

File: src/components/Dialogs/CommentFormBetaDialog/CommentForm/styles.module.css (hunk 2 above)
Comment (pitb2022) [reply src/components/Dialogs/CommentFormBetaDialog/CommentForm/styles.module.css:21]: Use global classname

**Pull request merged**